backend/app/tests/test_ai_processing_status_bug.py

```
@@ -37,9 +37,9 @@
         After fixing the bug, it should PASS.
         """
         # Arrange: Create a content item
-        content_item = Content(
-            id="test-content-id",
-            user_id="test-user-id", 
+        content_item = ContentItem(
+            id=uuid.uuid4(),
```

Ruff (0.12.2)

Undefined name `uuid`

(F821)